coalib/results/TextPosition.py

```diff
@@ -2,6 +2,10 @@
     enforce_signature, generate_ordering, generate_repr)
 
 
+class ZeroOffsetError(Exception):
```

Comment:
Should be subclassed from `ValueError`, esp. as you do checks on input values and raise it inside `__init__`